docs/standard/class-library-overview.md

```diff
@@ -108,9 +109,9 @@ ass objects|<xref:System.Object>|The root of the object hierarchy.|**Object**|**
   
```

Comment:
the previous two lines has one extra asterisk that is causing formatting issues in the rest of the text. perhaps the asterisk should come after the bold formatting like
`**String**\*` instead of `**String\***`

Reply:
The problem here is not only the formatting break, but I'm not sure what * is. (I know it's a pointer, but I'm not sure that String* works in C++.) I've changed the language label to C++/CLI. @ghogen, could you take a look at it for accuracy?

Reply:
@mikeblome can you help?